README.md

```diff
@@ -1079,6 +1079,20 @@ Here is how the `errors` package should be used:
    `errors.WithStackTrace`. This gives us a stacktrace as close to the source as possible.
 1. If you need to get back the underlying error, you can use the `errors.IsError` and `errors.Unwrap` functions.
 
+#### Formatting
+
+Every source file in this project should be formatted with `go fmt`. Feel free to accomplish this any way you choose. There are also few helper scripts and targets in the Makefile that can help with this (mostly taken from the [terraform repo](https://github.com/hashicorp/terraform/)):
```

Comment:
I would actually explicitly ask folks to install the precommit hook to automatically format their code.